python/tvm/testing/__init__.py

```diff
@@ -14,13 +14,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-[pytest]
-markers =
-    gpu: mark a test as requiring a gpu
-    tensorcore: mark a test as requiring a tensorcore
-    cuda: mark a test as requiring cuda
-    opencl: mark a test as requiring opencl
-    rocm: mark a test as requiring rocm
-    vulkan: mark a test as requiring vulkan
-    metal: mark a test as requiring metal
-    llvm: mark a test as requiring llvm
+"""Namespace for tvm testing utilities"""
+
+from .testing import *
```

Comment:
i agree with the reorg here but should we use this opportunity to split apart testing.py?

Reply:
I think that makes sense.  That would allow `testing.py` to have anything that is expected to be used externally, such as `tvm.testing.assert_allclose`, while `plugin.py` has all the interactions with the pytest framework itself.  I want to wait for #8542 to be resolved first to avoid annoying merge conflicts, but a wider split makes sense.

Reply:
ok sounds good to me.